eslint/babel-eslint-parser/src/convert/convertAST.js

```diff
@@ -1,14 +1,10 @@
 import { types as t, traverse } from "@babel/core";
 
 function convertNodes(ast, code) {
-  const state = { source: code };
   const astTransformVisitor = {
     noScope: true,
     enter(path) {
-      const node = path.node;
-
-      // private var to track original node type
-      node._babelType = node.type;
```

Comment:
I was wondering why we had this in the first place, but a quick search on GitHub ([search](https://github.com/search?p=1&q=_babelType&type=Code)) shows that it's only used in snapshot tests output: it can be safely removed.